Maddori.Apple/Maddori.Apple/Network/EndPoint/TeamDetailEndPoint.swift

```diff
@@ -12,6 +12,7 @@ enum TeamDetailEndPoint<T: Encodable>: EndPointable {
     case fetchTeamInformation
     case deleteTeam
     case fetchUserTeamList
+    case putEditProfile
```

Comment:
p2;

앞서 남긴 리뷰대로 수정하려면 여기도 수정해야합니다.

```suggestion
    case putEditProfile(T)
```